tests/test_strain_collection.py

```diff
@@ -43,14 +43,84 @@ def test_iter(collection: StrainCollection, strain: Strain):
 def test_add(strain: Strain):
     sut = StrainCollection()
     sut.add(strain)
+    assert len(sut) == 1
     assert strain in sut
     for name in strain.names:
         assert name in sut._strain_dict_name
 
 
-def test_remove(collection: StrainCollection, strain: Strain):
+def test_add_same_id_same_alias(strain: Strain, collection: StrainCollection):
+    collection.add(strain)
```

Comment:
General question about pytest: when you pass in a var like `strain`, type hinting that it's a `Strain` object (as above), does pytest instantiate it internally with the defaults? `collection` is defined above as a fixture, so it's clear, but `strain` it's not so I was wondering this

Reply:
The strain is a shared fixture defined in the `tests/conftest.py`.